stream-compositions/events/events/grandcentral/party-event.json

```diff
@@ -0,0 +1,284 @@
+                  "SOCIAL_SECURITY_NUMBER",
+                  "DRIVERS_LICENSE_NUMBER",
+                  "PASSPORT_NUMBER",
+                  "IDENTITY_CARD_NUMBER",
+                  "EMPLOYER_IDENTIFICATION_NUMBER"
+                ]
+              }
+            }
+          }
+        },
+        "personName": {
+          "type": "object",
+          "properties": {
+            "familyName": {
+              "type": "string"
+            },
+            "firstName": {
+              "type": "string"
+            },
+            "middleName": {
+              "type": "string"
+            }
+          }
+        },
+        "phoneAddresses": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "countryCode": {
+                "type": "string"
+              },
+              "countryIsoCode": {
+                "type": "string"
+              },
+              "key": {
+                "type": "string"
+              },
+              "number": {
+                "type": "string"
+              },
+              "primary": {
+                "type": "boolean"
+              },
+              "type": {
+                "type": "string",
+                "enum": [
+                  "MOBILE",
+                  "HOME",
+                  "WORK",
+                  "OTHER"
+                ]
+              }
+            }
+          }
+        },
+        "postalAddresses": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "addressLine": {
+                "type": "string"
+              },
+              "buildingNumber": {
+                "type": "string"
+              },
+              "country": {
+                "type": "string"
+              },
+              "countrySubDivision": {
+                "type": "string"
+              },
+              "department": {
+                "type": "string"
+              },
+              "key": {
+                "type": "string"
+              },
+              "postalCode": {
+                "type": "string"
+              },
+              "primary": {
+                "type": "boolean"
+              },
+              "streetName": {
+                "type": "string"
+              },
+              "subDepartment": {
+                "type": "string"
+              },
+              "townName": {
+                "type": "string"
+              },
+              "type": {
+                "type": "string",
+                "enum": [
+                  "BUSINESS",
+                  "CORRESPONDENCE",
+                  "POSTAL",
+                  "PO_BOX",
+                  "RESIDENTIAL",
+                  "PERMANENT",
+                  "TEMPORARY"
+                ]
+              }
+            }
+          }
+        },
+        "preferredLanguage": {
+          "type": "string"
```

Comment:
This could do with a specification... is it "English" or "en" (or "en_US")
We have a similar confusion in ModelBank/Mambu: https://github.com/baas-devops-reference/arrangement-pull-integration-service/blob/master/src/main/java/com/backbase/productled/mapper/CountryMapper.java

Reply:
as we are not updating any user profile here, this field is completely ignored so far